### DISCUSSION TITLE
Integrated with backup unit meter doesn't show correct values on imported (and potentially exported) kwh

### DISCUSSION THREAD

**Discussion by miqm** [binsentsu/home-assistant-solaredge-modbus#215] [Q&A] [answered]
Hello,

I integrated this component into my HA. Everything looks good, however for the imported energy meter values seems... odd. I've meter integrated in a backup unit (meter is not an independent one but it's put inside backup box - apologise for not being technically precise, this is a quite new area for me).
Anyway, AC power seems ok and matches current usage shown on SE portal - but the imported KWH stays at same level and due to the current weather conditions majority of energy is being imported. Also - I don't know about exported KWH - as it's to cloudy to export anything.

![image](https://github.com/binsentsu/home-assistant-solaredge-modbus/assets/7921224/c075e968-4ed5-4151-a169-6aa6b31203b0)

Any help appreciated.

**Comment by miqm** [comment, marked as answer]
Resolved by updating backup box firmware

**Reply by SimonX200** [reply]
I wonder how did you got the update? 

I have the same problem: strange readings and values shown in mySolarEdge. Daily kWh breakdown is extrem off.  Readings from HA look 30% off still, but better than the kWh in the app. Had opened a ticket November last year, but I didnt continued to exchange useless mails with level1.

**Reply by miqm** [reply]
My installer downloaded new firmware though setapp. I spoke with him recently and since then he had similar case also with backup box meter.

Although my modbus readings are fine, the values in my SolarEdge are rubbish. For each day they're ok, but for weeks are months they don't add up.
